### DISCUSSION TITLE
Match parent surround character

### DISCUSSION THREAD

**Discussion by ok-nick** [helix-editor/helix#7993] [Q&A]
If I have code like this:
```lua
{
    {
        foo = "bar", -- and my cursor is here
    },
}
```
Can I move to the parent match by pressing a key, such as `n` and back to the child match by pressing a key like `N`?

If this isn't a feature or an existing issue, I would be happy to create one.

**Comment by ghost** [comment]
~`mm` can move to the matching bracket.~

whoops, misunderstood your question.
